content/team/anibalsolon.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Anibal Solon Heinsfeld"
+date: 2022-01-26T17:32:40Z
+draft: false
+position : "Graduate Research Assistant"
+description : "Anibal is a graduate research assistant in the brainlife.io project. His focus is Machine Learning methods applied in Neuroscience, developing methods that account for reproducibility and reliability. Besides his academic career, Anibal contributed to several open-source projects and loves computer infrastructure and programming. He joined the University of Texas at Austin in 2019 and joined the brainlife.io team in 2021."
+image : "/images/avatar/anibalsolon.jpg"
+tags : []
```

Comment:
tags: ['active']

This will add you to the active team :)

Reply:
Thanks! I've included the tag.